GPL/gpl.gradle

```diff
@@ -45,6 +46,11 @@ ext.getCurrentPlatformName = {
 			return 'osx64'
 		}
 	}
+	else if (osName.startsWith("Mac OS X")) {
+		if (isAARCH_64) {
```

Comment:
This (line 50-52) should be changed to an "else if" after the "if (isX86_64)" conditional from line 45-47 I think?

Reply:
Sure thing, looks neater now, thanks!